test/remote/db_tests.js

```diff
@@ -66,6 +66,13 @@ describe('remote db', function() {
       })
   })
 
+  beforeEach(() => {
+    return db.createAccount(ACCOUNT)
```

Comment:
Do we want `before` instead of `beforeEach`, maybe? Is there benefit in creating and deleting the same account for each test?